### PR TITLE
Playground: Added the pop the balloons game to the playground allthethings level

### PR DESCRIPTION
**Pull request by jmkulwik**
This adds the "Pop the balloons" game as an allthethings level for the playground mini app. Currently, nothing happens except some console output, but as the Playground API is built, more of the game will be enabled.

image

Additionally, this PR adds a little more clarity to the submittable allthethings Java Lab level so it is more differentiated from the first Java Lab allthethings level.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/scripts/levels/Allthethings Java Lab Playground.level [hunk 1]
```diff
@@ -2,15 +2,67 @@
...
+    "long_instructions": "This is a sample Playground level. It responds to clicks on the images that are in the program.",
     "mini_rubric": "false",
     "hide_share_and_remix": "false",
-    "csa_view_mode": "playground"
+    "csa_view_mode": "playground",
+    "submittable": "false",
+    "start_sources": {
+      "Dice.java": {
+        "text": "import org.code.playground.*;\nimport java.io.FileNotFoundException;\n\n// LESSON 1\npublic class Dice {\n\n  private ImageItem[] diceImages;\n\n  public Dice(String[] diceFiles, int xPos, int yPos) throws FileNotFoundException {\n    diceImages = new ImageItem[6];\n    \n\tfor (int i = 0; i < diceFiles.length; i++) {\n      diceImages[i] = new ImageItem(diceFiles[i], xPos, yPos,  50, 50);\n    }\n  }\n\n  public void displayDice(int value) {\n    Playground.board.addItem(diceImages[value - 1]);\n  }\n\n  public ImageItem getDice(int value) {\n    return this.diceImages[value - 1];\n  }\n\n  public void hideDice() {\n    for (ImageItem diceImage : this.diceImages) {\n    \tPlayground.board.removeItem(diceImage);\n    }\n  }\n}",
```
Comment (molly-moen) [inline dashboard/config/scripts/levels/Allthethings Java Lab Playground.level:12]: We had a couple changes to the api we'll need to reflect here:
- we got rid of `addItem` and replaced it with `addTextItem` and `addImageItem`
- `board.exit()`now throws a `PlaygroundException`

Comment (sanchitmalhotra126) [reply dashboard/config/scripts/levels/Allthethings Java Lab Playground.level:12]: On that note, for my understanding, how do we update these files when we have API changes? Do we need to manually copy the code into this level file, or is this autogenerated somehow?

Comment (molly-moen) [reply dashboard/config/scripts/levels/Allthethings Java Lab Playground.level:12]: it's manual :(

Comment (jmkulwik) [reply dashboard/config/scripts/levels/Allthethings Java Lab Playground.level:12]: It is a manual process, but you don't have to edit the file directly. If you go to the level's url and are logged in with levelbuilder permissions, you can click the "start" link in the "extra links" menu. Any code added to the level will be saved to the level when the "save" button is clicked.

**Pull request closed without merging**